starknet_py/hash/transaction.py

```diff
@@ -76,6 +78,38 @@ def compute_transaction_hash(
     )
 
 
+def compute_invoke_transaction_hash(
+    *,
```

Comment:
Do we want this function to have keyword-only arguments? It's a bit chaotic in this file.

Reply:
Ideally everything should be keyword only 😃 

I decided to go with keyword-only as the order of the parameters is different in many of these functions.
Starknet.js has them ordered similiarly (the order in which these arguments are, actually) in each of their functions, but unfortunately we cannot change them without making it a breaking change.
Also, keyword only functions are less error prone.

As for the file, it contains just the methods that are for computing a hash of a transaction.
Since all functions but for Invoke were present here, I don't think it should be somewhere else.